Android/ChorusRFLaptimer/app/src/main/res/layout/race_result_list_group.xml

```diff
@@ -64,18 +66,18 @@
                         android:id="@+id/textLabel0"
                         android:layout_width="wrap_content"
                         android:layout_height="wrap_content"
-                        android:text="Race time: "
                         android:gravity="center_vertical"
-                        android:textSize="12sp"/>
+                        android:text="@string/race_time"
+                        android:textSize="12sp" />
 
                     <TextView
                         android:id="@+id/textTotalRaceTime"
                         android:layout_width="0dp"
                         android:layout_height="wrap_content"
-                        android:text="Text"
+                        android:layout_weight="1"
                         android:gravity="center_vertical"
                         android:textSize="12sp"
-                        android:layout_weight="1" />
+                        tools:text="2:12" />

```

Comment:
can you please clarify what's this? :)

Reply:
tools:text is the TextView text that is gonna be applied in IDE's "Designer mode" without setting the text in runtime. That way you can play around in design mode with different values, 2:12 means like 2 minutes and 12 seconds